x-pack/plugins/security_solution/public/detections/containers/detection_engine/exceptions/get_query_filter.ts

```diff
@@ -10,21 +10,21 @@ import type {
   ExceptionListItemSchema,
   CreateExceptionListItemSchema,
 } from '@kbn/securitysolution-io-ts-list-types';
-import { buildExceptionFilter } from '@kbn/securitysolution-list-utils';
 import type { Filter, EsQueryConfig, DataViewBase } from '@kbn/es-query';
 import { buildEsQuery } from '@kbn/es-query';
 
-import type { ESBoolQuery } from '../typed_json';
-import type { Query, Index } from './schemas/common/schemas';
+import type { Query, Index } from '../../../../../common/detection_engine/schemas/common';
+import type { ESBoolQuery } from '../../../../../common/typed_json';
+import { getExceptionFilterFromExceptions } from './api';
 
-export const getQueryFilter = (
+export const getQueryFilter = async (
```

Comment:
A more descriptive name might be helpful here, there are a number of `getQueryFilter` functions in the codebase and we're splitting ours into 2 separate implementations now as well.